lib/storage/commands/delete.rb

```diff
@@ -33,8 +33,9 @@ def run
         # ARGS:
         # [ target_file ]
         
+        args[0] = args[0].gsub(%r{/+}, "/")
```

Comment:
Please could you set a new variable here and pass that to `client.delete` instead?  It's bad practice to mutate the `args` array. Same with the other examples here.